### ISSUE TITLE
[v2] Limit query by placing caret inside operation not working

### ISSUE THREAD

**Issue by arielnmz** [jimkyndemeyer/js-graphql-intellij-plugin#243]
## Version and Environment Details
__Operation system__: 

```
$ cat /etc/fedora-release&& uname -r
Fedora release 29 (Twenty Nine)
5.0.4-200.fc29.x86_64
```

__IDE name and version__: 
Pycharm PRO 2019.1

__Plugin version__: 
2.0.0

## Expected Behaviour
Placing caret inside an operation and pressing Ctrl+Enter should send only said operation

## Actual Behaviour
Query fails:

```
{
  "errors": [
    {
      "message": "Must provide operation name if query contains multiple operations."
    }
  ]
}
```
Presumably because it's sending everything in the graphql file

## Steps to Reproduce / Link to Repo with Reproduction and Instructions

1. Create a new `.graphql` file and add at least two operations:

```
query query1 { ... }
query query2 { ... }
```
2. Place the caret in any place of any operation and press Ctrl+Enter
3. Operation fails

Note: Selecting and pressing Ctrl+Enter works just fine.

**Comment by jimkyndemeyer** [comment]
Hi Benjamín.

Thanks for using the plugin.

Yeah, this didn't make it into the 2.0.0 even though 1.x has it. As you said, the workaround is to select the operation in the editor even though that's not as elegant.

I hope to have this feature back in 2.1.0.

Best regards,
Jim.

**Comment by jimkyndemeyer** [comment]
I've implemented this for the 2.1.0 release that's scheduled for mid-next week.

The contextual queries are now also capable of including referenced fragments that are declared in other files.

Best regards,
Jim.

**Comment by akahan** [comment]
Doesn't work on new version of Idea (IntelliJ IDEA 2020.3.2 (Ultimate Edition) Build #IU-203.7148.57).
When I want to send one of the operation, I put a caret on that operation, but the whole file is sent.

PS: I confirm that if you select the required operation, only that operation is sent. But this is very inconvenient.
